src/core/Akka.FSharp/FsApi.fs

```diff
@@ -481,6 +480,74 @@ let asyncReceive (i : Inbox) : Async<'Message option> =
     }
 
 /// <summary>
-/// Orders inbox to watch an actor targeted by provided <paramref name="actorRef"/>.
+/// Orders a <paramref name="watcher"/> to monitor an actor targeted by provided <paramref name="subject"/>.
+/// When an actor refered by subject dies, a watcher should receive a <see cref="Terminated"/> message.
+/// </summary>
+let monitor (subject: ActorRef) (watcher: ICanWatch) : ActorRef = watcher.Watch subject
```

Comment:
Following typical F# library designs, I would recommend making "Akka.FSharp" a namespace, and put all these let-bindings in a "AutoOpen" module.  This reduces the nesting of types in the compiled IL which can help in quite a number of situations.